Subsurface/Subsurface/SAppDelegate.m

```diff
@@ -21,6 +21,7 @@
 #import "SAppDelegate.h"
 
 #define kAnimationOpacityKey    @"animateOpacity"
+#define HOUR 3600
```

Comment:
Let's stick to a way of naming defines as in other places among this codebase — starts with lowercase "k" and then a CamelCase